src/site/content/en/blog/fast-ads-matter/index.md

```diff
@@ -0,0 +1,196 @@
+  <figcaption class="w-figcaption">
+    Source: Google Internal Data, December 2016 to January 2017.
+  </figcaption>
+</figure>
+
+## A framework for thinking about ad speed
+
+Modern websites tend to have complex and diverse ad serving setups, which means
+there's no one-size-fits-all method of making ads fast. Instead, the following
+sections give you a framework for thinking about ad speed. Some points are
+specific to Google Ad Manager, but the principles apply even if you're using a
+different ad server.
+
+### Know why you want to improve ad speed
+
+Before you start working to improve ad speed, you should be clear on what your
+goals are. Is it to improve the user experience? To increase viewability? Both?
+
+Whatever your specific goals are, it's important to identify the metrics you can
+use to measure and track progress towards them over time. Having the right
+metrics in place allows you to:
+
+*   Know if the changes you're making are moving you in the right direction.
+*   Run experiments, such as A/B tests, to evaluate the effectiveness of
+    specific changes.
+
+Once you've decided on the metrics that make sense for you, be sure to configure
+reporting so you can easily keep track of them. A dashboard you can check
+periodically or scheduled reports sent to you by email work well for that.
+
+### Know your inventory and dependencies
+
+To identify opportunities for improving ad speed, you first need to understand
+the types of inventory your site supports and the technical dependencies of
+each.
+
+As an example, suppose a site supports the following inventory types:
+* Desktop leaderboard
+* Mobile banner
+
+To load and display ads, the example site uses the following:
+* A consent management platform
+* Audience scripts
+* Header bidding scripts
+* A rendering framework
+
+First, create a flowchart for each inventory type to visualize how the various
+dependencies interact in order to load and display an ad. Desktop leaderboard
+inventory may look like this:
+
+![An example workflow for the desktop leaderboard inventory
+type.](./desktop-leaderboard.svg)
+
+While a more complex inventory type, such as mobile banner, may look like this:
+
+![An example workflow for the mobile banner inventory
+type.](./mobile-banner.svg)
+
+Then, use this information to create a simple table like the one below, which
+maps each inventory type to its dependencies in an easily digestible format.
+
+| Type of inventory   | Consent management platform | Audience script    | Header bidding script | Rendering framework |
+|:--------------------|:---------------------------:|:------------------:|:---------------------:|:-------------------:|
+| Desktop leaderboard |  &#x2714;                   | &#x2714; (X)       | &#x2714; (A)          | -                   |
+| Mobile banner       |  &#x2714;                   | &#x2714; (X and Y) | &#x2714; (A and B)    | &#x2714;            |
+
```

Comment:
@mihajlija I think we usually avoid tables with a lot of columns because they don't display well on mobile. I'm OK with it on this post but it should probably be avoided in the future.

Reply:
Markdown syntax for tables is also hard to read. We should probably just use HTML syntax going forward.